logger/log.go

```diff
@@ -112,7 +112,5 @@ func (l *Logger) Debugf(format string, v ...interface{}) {
 }
 
 func (l *Logger) Tracef(format string, v ...interface{}) {
-	if l.trace == true {
```

Comment:
I believe this check is safe to remove since AFAICT all of the calls to this already assume tracing is enabled, either globally or per-client.